BedrockServer.cpp

```diff
@@ -1121,6 +1121,7 @@ BedrockServer::BedrockServer(const SData& args_)
     _lastQuorumCommandTime(STimeNow()), _pluginsDetached(false), _socketThreadNumber(0),
     _outstandingSocketThreads(0), _shouldBlockNewSocketThreads(false)
 {
+    _isCommandPortLikelyBlocked = false;
```

Comment:
Does this need the mutex lock for the port threads? Probably not since we're initializing it.

Reply:
I tried to add this to the initializers above but it wouldn't compile, probably because its an atomic type

Reply:
actually I can, my b 1 sec